zap/src/main/java/org/parosproxy/paros/extension/history/ExtensionHistory.java

```diff
@@ -463,13 +464,19 @@ public void run() {
         }
     }
 
-    private void searchHistory(HistoryFilter historyFilter) {
+    /**
+     * Returns a sorted list of History IDs for the visible history references.
+     *
+     * @return a list of History IDs for the visible history references.
+     * @since 2.13.0
+     */
+    public List<Integer> getHistoryIds() {
```

Comment:
- Is this public only because of the context menu?
- `getHistoryIdsOfHistType` by contract does not guarantee ordering. The method says the list is sorted but then it's being sorted in the context menu.

Reply:
Doh, I had actually relocated the sort to here last night but apparently didn’t push it ☹️

Reply:
Yes it's public only because of the context menu, though it could be needed by neonmarker or other code.